es/news/_posts/2015-12-11-ruby-2-3-0.preview2-released.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: news_post
+title: "Liberada la versión 2.3.0-preview2 de Ruby"
+author: "naruse"
+translator: "mariochavez"
+date: 2015-12-11 14:00:00 +0000
+lang: es
+---
+
+Nos complace anunciar la entrega de Ruby 2.3.0-preview1.
+
+Ruby 2.3.0-preview2 es el segundo preview de la serie 2.3.0.
+Incluye muchas mejoras y nuevas características.
+
+[Frozen String Literal Pragma](https://bugs.ruby-lang.org/issues/11473) ha sido 
+introducida. Con Ruby 2.1, `"str".freeze` está optimizado para reducir la 
+creación de nuevos objetos. Ruby 2.3 introduce un nuevo comentario _mágico_ y
+una opción de línea de comando para especificar todas las literales de
+cadena de texto.
+Adicionalmente para depurado, usted puede obtener donde el objeto fue creado
```

Comment:
"usted" is too formal... change the addressing to "tú". 

`Aparte, para depurado puedes obtener donde fue creado el objeto...`